cluster/examples/workloads/kubernetes/wordpress-aws/app.yaml

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
It would really be nice to highlight that this is the same exact file in the `wordpress-gcp` file.

Reply:
I had a minor difference between the two copies of app.yaml which bit me.  The generated sql secret landed in a different namespace with a different name. 

I'll go for a de-duplication.  A symlink is one approach, but I'll shoot for:

```
cluster/examples/workloads/kubernetes/wordpress/app.yaml
cluster/examples/workloads/kubernetes/wordpress/gcp/cluster.yaml
cluster/examples/workloads/kubernetes/wordpress/gcp/provider.yaml
cluster/examples/workloads/kubernetes/wordpress/aws/cluster.yaml
cluster/examples/workloads/kubernetes/wordpress/aws/provider.yaml
```

Unless, it would be better to specify the provisioner in each `app.yaml` to disambiguate for anyone running the demo on all supported clouds. What do you think @negz ?

Reply:
What do you think about maintaining one set of providers for all examples? I was thinking of going that direction with the updates I need to make to claim examples.

Reply:
I like it.

Some of the resourceclasses may not be needed by the various examples, but it is probably simpler (better) to get this out of the way in a single document and yaml.

The aws-credentials.sh script differs from the gcp-credentials.sh script in that it isn't really creating credentials, it is pre-provisioning.  If the aws-provider.md doc took the stance of configuring credentials, pre-provisioning, and configuring the Crossplane provider and resources classes, the actual examples themselves would become much easier to document.

We can attack that after these PRs are merged independently as part of #519.